pkgs/top-level/perl-packages.nix

```diff
@@ -18833,6 +18833,13 @@ with self; {
     };
     buildInputs = [ TestDeep TestDifferences TestException TestMost TestWarn ];
     propagatedBuildInputs = [ CDDBFile Moo ];
+    patches = [
+      # https://rt.cpan.org/Public/Bug/Display.html?id=152242
+      (fetchpatch2 {
+        url = "https://salsa.debian.org/perl-team/modules/packages/libnet-freedb-perl/raw/master/debian/patches/implicit-function-declaration.patch";
```

Comment:
This patch URL is unstable:
```
┃ error: hash mismatch in fixed-output derivation '/nix/store/vjpscs415gsschikixzyxrcc75q0vkkp-implicit-function-declaration.patch.drv':
┃         likely URL: https://salsa.debian.org/perl-team/modules/packages/libnet-freedb-perl/raw/master/debian/patches/implicit-function-declaration.pat…
┃          specified: sha256-0wV/wpTnNbXYkxvvfUrfz0vFUio5wuE0o1+VyXpe51I=
┃             got:    sha256-oIfJuLzTm0JLzAUvP1mcGCm6oWGpcLWDPqHN/4pKC9Q=
```

Reply:
Sorry.. did not realize that.

I think this is a better source: https://sources.debian.org/data/main/libn/libnet-freedb-perl/0.10-5/debian/patches/implicit-function-declaration.patch as similar source is used in other pearl packages within this file.